### PR TITLE
[Trivia] New lists and fixes

### PR DESCRIPTION
**Pull request by MiniJennJenn**
### Type

- [ ] Bugfix
- [x] Enhancement
- [x] New feature

### Description of the changes

Updated: 
-
 - leagueoflegends
 - leagueults

New: 
-
 - games (thanks bobloy for submitting boardgames but since most of the questions weren't "boardgames" I combined it with the pre-existing games list)
 - sports (some of the questions from the aforementioned boardgames list went into this category as well)
 - r6seige (thanks MHLoppy)

### CODE REVIEW

**Comment by palmtree5** [comment]
I'm noticing there's curly quotes in at least the games trivia list atm

**Comment by MiniJennJenn** [comment]
Could you give me an example of where? I don't see any.

File: redbot/cogs/trivia/data/lists/games.yaml [hunk 1]
```diff
@@ -1,366 +1,577 @@
...
+If you "peg out" what game are you playing?:
+- Cribbage
+If you're killing a goomba, what game are you playing?:
+- Super Mario Bros
+- Mario
+"In \"Animal Crossing: Wild World\", what wildlife is found in the town observatory?":
+- Owl
+"In \"GTA: San Andreas\", what is CJ's older brother's name?":
+- Sweet
+In "Twisted Metal 2″, what does Krista Sparks drive?:
```
Comment (palmtree5) [inline redbot/cogs/trivia/data/lists/games.yaml]: Here

**Comment by hatred2k** [comment]
Not sure if this is the correct issue, however, I've updated the sports trivia to accommodate for the most recent championships as well as I've added a couple of new ones.
V3: [sports.yaml](http://hatred2k.com/assets/sports.yaml)

**Review by Drapersniper — CHANGES_REQUESTED** [review]
@MiniJennJenn  could you please fix the conflicts so we can get this reviewed?

**Comment by MiniJennJenn** [comment]
So, I can't actually access that link from hatred.

**Comment by hatred2k** [comment]
> So, I can't actually access that link from hatred.

Updated version **[here](https://pastebin.com/raw/RcPZ02A7)**.

**Comment by MiniJennJenn** [comment]
Okay. The rest of this is fine for commit. I think we just commit this one and I'll create a new PR for hatred's newer sports trivia version.

**Comment by MiniJennJenn** [comment]
@hatred2k In the future, if you're going to update your trivia/add more, I suggest updating from the one we have. I had fixed it in the past to resolve a number of typos. Since your version doesn't have those fixes, I'll need to adjust accordingly.

**Comment by MiniJennJenn** [comment]
Closing. All fixed in #4026

**Pull request closed without merging**